neard/src/runtime/errors.rs

```diff
@@ -74,6 +75,12 @@ impl QueryError {
             node_runtime::state_viewer::errors::ViewStateError::InternalError { error_message } => {
                 Self::InternalError { error_message, block_height, block_hash }
             }
+            ViewStateError::AccountDoesNotExist { requested_account_id } => {
+                Self::InvalidAccount { requested_account_id, block_height, block_hash }
+            }
+            ViewStateError::AccountStateTooLarge { requested_account_id } => {
+                Self::InvalidAccount { requested_account_id, block_height, block_hash }
```

Comment:
@frol @khorolets should this be a different error or is it fine to reuse `InvalidAccount`?

Reply:
Let's use a separate error variant to properly communicate the reason. Following the style of InvalidAccount, UnknownAccount NoContractCode, I suggest TooLargeContractState